gpm_api/tests/io/test_checks.py

```diff
@@ -0,0 +1,478 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jul 17 15:41:14 2023
+
+@author: ghiggi
+"""
+
+import pytest
+import datetime
+import numpy as np
+from gpm_api.io import checks
+from gpm_api.io.products import available_products, available_scan_modes
+
+
+def test_is_not_empty() -> None:
+    ''' Test is_not_empty() '''
+
+    # Test a non empty object
+    res = checks.is_not_empty([1, 2, 3])
+    assert res is True, "Function returned False, expected True"
+
+    # Test an empty object
+    res = checks.is_not_empty([])
```

Comment:
Can you please also add the test for `None` , `False`,  `True`, `()`, `{}`.
I don't expect behaviors to change ... but ...

Reply:
How would you like the test of `False`, `True`, and `None` to behave here? 

`False == False`, `True == True`, `None=False`? I only ask because they are objects that can be considered non-empty

Reply:
@ evanjt  `None --> False`. I think nowhere in the code we do `is_empty(True)` or `is_empty(False)` or `is_empty(2)`. 
I wonder if we should maybe avoid using `is_empty` and `is_not_empty` in the code ... since might cause more confusion than code clarity. 
In a if clause: None, (), {}, [] would be a `False`,  and the rest a `True` ... 
If you get a `np.bool` or np.array(False) what `is_empty` return?